chia/data_layer/data_layer_wallet.py

```diff
@@ -1266,17 +1266,17 @@ async def finish_graftroot_solutions(offer: Offer, solver: Solver) -> Offer:
                 except EvalError:
                     new_spends.append(spend)
                     continue
-                mod, curried_args = graftroot.uncurry()
+                mod, curried_args_prg = graftroot.uncurry()
                 if mod == GRAFTROOT_DL_OFFERS:
-                    _, singleton_structs, _, values_to_prove = curried_args.as_iter()
+                    _, singleton_structs, _, values_to_prove = curried_args_prg.as_iter()
```

Comment:
`curried_args` is a `Program` here, but in other places it's a `List[Program]` in other places, so it needs a different name here